### PR TITLE
kill noneulized_users report

### PR DESCRIPTION
**Pull request by dannyroberts**
it already wasn’t linked to anywhere in the UI and no one’s accessed it
since datadog was set up

it has also been broken since the users migration because of a typo
(and it was never cached despite an apparent attempt because of two typos)

### CODE REVIEW

File: corehq/couchapps/__init__.py [hunk 1]
```diff
@@ -5,7 +5,6 @@
     'form_question_schema': 'meta',
     'receiverwrapper': 'receiverwrapper',
     'users_extra': (settings.USERS_GROUPS_DB, settings.NEW_USERS_GROUPS_DB),
-    'noneulized_users': (settings.USERS_GROUPS_DB, settings.NEW_USERS_GROUPS_DB),
```
Comment (dannyroberts) [inline corehq/couchapps/__init__.py]: my typo—should have been `eula_report` (so this hasn't worked in a while)

File: corehq/apps/cachehq/cachemodels.py [hunk 2]
```diff
@@ -29,7 +29,6 @@ class UserGenerationCache(GenerationCache):
         "users/mailing_list_emails",
         "domain/old_users",
         "users_extra/phones_to_domains",
-        "eula_reports/non_eulized_users"
```
Comment (dannyroberts) [inline corehq/apps/cachehq/cachemodels.py]: ancient typo :) should have been `eula_report/noneulized_users`

**Comment by dannyroberts** [comment]
and of course this kills `eula_report/noneulized_users` view in the main db (should have already been in the users db but for my typo!)

**Comment by czue** [comment]
did you check with @dimagi/product on this? i have the vague sense that they may have had some periodic workflow around it but guessing that's not true anymore.

**Comment by czue** [comment]
:+1: from me if/after product signs off

**Comment by amsagoff** [comment]
This was part of an old workflow of Carter's to make sure everyone signed
the EULA when we first deployed it. I'll check with him and report back.

**Comment by dannyroberts** [comment]
@amsagoff says KILL

**Comment by amsagoff** [comment]
:skull:

**Comment by dannyroberts** [comment]
Ok I'm gonna self merge based on cory and amelia's comments

**Comment by czue** [comment]
:rage1: :skull:

**Pull request merged**